client/tests/input_selection/alias_outputs.rs

```diff
@@ -1993,3 +1996,47 @@ fn remainder_address_in_governor() {
         }
     });
 }
+
+#[test]
+fn do_not_change_amount_of_governance_transition() {
+    let protocol_parameters = protocol_parameters();
+    let alias_id_1 = AliasId::from_str(ALIAS_ID_1).unwrap();
+
+    let inputs = build_inputs(vec![Alias(
+        2_000_000,
+        alias_id_1,
+        0,
+        BECH32_ADDRESS_ED25519_0,
+        BECH32_ADDRESS_ED25519_1,
+        None,
+        None,
+        None,
+        None,
+    )]);
+    let outputs = build_outputs(vec![Basic(
+        1_000_000,
+        BECH32_ADDRESS_ED25519_0,
+        None,
+        Some(BECH32_ADDRESS_ED25519_1),
+        None,
+        None,
+        None,
+        None,
+    )]);
+
+    let selected = InputSelection::new(
+        inputs,
+        outputs,
+        addresses(vec![BECH32_ADDRESS_ED25519_0]),
+        protocol_parameters,
+    )
+    .select();
+
+    assert!(matches!(
+        selected,
+        Err(Error::InsufficientAmount {
+            found: 2_000_000,
+            required: 3_000_000,
+        })
+    ));
```

Comment:
I think that's wrong or at least confusing
Maybe we should store the amount of provided outputs and also return that, because only a basic output with 1_000_000 was provided, why is now 3_000_000 required? 3_000_000 makes sense if you need to transition the alias, but if we can't use any amount, why is it even added to the `found`?

Reply:
Yeah but the alias is actually required because of the basic sender

Reply:
That's the only way to get an automatic governance transition

Reply:
Ah missed the sender thing